ipgeo/nexttrace_api_v4_test.go

```
@@ -209,6 +278,282 @@ func TestLeoIPNextTraceAPIV4HTTPReusesCachedClientAndConnection(t *testing.T) {
+func TestDialNextTraceAPIV4FallsBackWhenCachedFastIPFails(t *testing.T) {
+	oldCache := util.GetFastIPCache()
+	oldMeta := util.GetFastIPMetaCache()
+	t.Cleanup(func() {
+		util.SetFastIPCacheState(oldCache, oldMeta)
+	})
+
+	listener, err := net.Listen("tcp", "127.0.0.1:0")
+	if err != nil {
+		t.Fatalf("net.Listen() error = %v", err)
+	}
+	defer listener.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `listener.Close` is not checked

(errcheck)